### PR TITLE
Assignment3 @ZackWard-student

### PR DESCRIPTION
**Pull request by sammyboy2000**

### CODE REVIEW

**Comment by sammyboy2000** [comment]
StaticWebApp:
https://icy-stone-0b3d2a71e.1.azurestaticapps.net/
API:
https://inconceivable-api.azurewebsites.net/swagger/index.html

**Comment by sammyboy2000** [comment]
API is currnetly broken on deployment, working tirelessly to fix it.

**Comment by hewrtarkhany** [comment]
looking good, I couldn't get any data back

**Comment by Disciple4GIT** [comment]
Everything looks really good! The only thing I noticed was the game didn't submit player data to the leaderboard. Besides that, everything seems to be in order

File: wordle-web/components/keyboard.vue [hunk 1]
```diff
@@ -1,112 +1,87 @@
 <template>
-  <v-card class="ma-0 pa-0" color="transparent" flat>
-    <v-row v-for="(charRow, i) in chars" :key="i" justify="center">
-      <v-col v-for="char in charRow" :key="char" cols="1" class="ma-0 pa-0">
-        <v-container class="text-center ma-0 pa-0">
+  <v-card class="my-5 pa-5">
```
Comment (mmwoodfo) [inline wordle-web/components/keyboard.vue:2]: Make sure that changes in your PR are only relevant to that of the Assignment.

File: wordle-web/components/player-info.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <v-card justify-center class="p-6">
+    <v-card>
+      <v-btn id="playerName" max-width="128" @click="toggleDialog">{{
+        getName()
```
Comment (mmwoodfo) [inline wordle-web/components/player-info.vue:5]: This should update as the player types and changes their name. You can do this by referencing the variable the v-text-field v-model references (playerName) as discussed in class.

**Review by mmwoodfo — COMMENTED** [review]
Make sure to add a link to your azure app in your PR comments somewhere.

**Pull request closed without merging**